ably/realtime_presence_integration_test.go

```diff
@@ -97,19 +91,28 @@ func TestRealtimePresence_Sync250(t *testing.T) {
 	assert.NoError(t, err,
 		"rg.Wait()=%v", err)
 
-	members2 := make([]*ably.PresenceMessage, 250)
+	members2 := make([]*ably.PresenceMessage, 0)
 	tout := time.After(250 * ablytest.Timeout)
+	client_ids := make(map[string]struct{})
 
-	for i := range members2 {
+	for len(client_ids) < 250 {
 		select {
 		case msg := <-sub2:
-			members2[i] = msg
+			members2 = append(members2,  msg)
+			client_ids[msg.ClientID] = struct{}{}
 		case <-tout:
 			t.Fatalf("waiting for presence messages timed out after %v", 250*ablytest.Timeout)
 		}
 	}
 
 	err = contains(members2, clients...)
+
+	if err != nil {
+		for k, v := range members2 {
+			fmt.Println(k, v)
+		}
+	}
+
```

Comment:
I can see `members2` is printed on line 116 as part of the assertion if this test fails. Is this a debug print that was left in and should be taken out?

Reply:
Yes probably. I wrote this before the summit and forgot about it.

Reply:
No worries, please delete the first print of `members2` and I'm happy to approve 👍